python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -998,21 +1026,21 @@ def fetch_row_counts(
         except ImportError:
             pass
 
-        def _threadpool_fetch_and_attach_row_count_metadata() -> (
+        def _threadpool_wrap_map_fn() -> (
             Iterator[Union[Output, AssetMaterialization, AssetObservation, AssetCheckResult]]
         ):
             with ThreadPoolExecutor(
                 max_workers=self._dbt_cli_invocation.postprocessing_threadpool_num_threads,
-                thread_name_prefix="fetch_row_counts",
+                thread_name_prefix=fn.__name__,
```

Comment:
Hmm we would have to check this: I'm wondering what would happen if the user passes in a lambda, rather than a declared function. We might have to manage the name on their behalf or something.

Reply:
Added a prefix to the fn name which at least points to this being the mapping fn, but not sure what else we can do here. If a user ends up defining many custom lambda postprocessing steps and has to figure out which isn't working and can't figure it out from stacktrace/other context they can always give them names.